pkg/config/config.go

```diff
@@ -49,3 +50,28 @@ func GetDefaultConfigPath() (string, error) {
 
 	return filepath.Join(home, DefaultConfigPath, DefaultConfigFileName), nil
 }
+
+func GetDefaultPath(fileName string) (string, error) {
```

Comment:
This new function is a duplicate with `GetDefaultConfigPath`

Reply:
Lets refactor `GetDefaultConfigPath` to get the desired location instead.

Reply:
Just had to change the name of the file, because 'DefaultConfigName' is set to 'config' and it brakes the config file.

Reply:
Yea, but we still have duplicated code in this function. Instead refactor `GetDefaultConfigPath`.